README.md

```diff
@@ -136,10 +144,10 @@ Client client = Clients.builder()
     .setClientId("{clientId}")
     .setScopes(new HashSet<>(Arrays.asList("okta.users.read", "okta.apps.read")))
     .setPrivateKey("/path/to/yourPrivateKey.pem")
-    // (or) .setPrivateKey("full PEM payload")
-    // (or) .setPrivateKey(Paths.get("/path/to/yourPrivateKey.pem"))
-    // (or) .setPrivateKey(inputStream)
-    // (or) .setPrivateKey(privateKey)
+    // (or) .setPrivateKey("full PEM payload");
```

Comment:
Do we really need a semicolon symbol here?

Reply:
🤣 
@sergiishamrai-okta It is your changes from here: https://github.com/okta/okta-sdk-java/blob/4170c6847e1ecbc88639a23ee9c424c87af67920/examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java#L80